docs/tmp-env-setup.md

```diff
@@ -0,0 +1,29 @@
+With the new microservices shipping, the old [environment-setup](environment-setup) document might not work. In the transition, this document may be helpful for developers. Here are some instructions for different scenarios, they are based on [next.config.js](../src/web/next.config.js) code logic. This `next.config.js` file configures which `env` file to use. It reads `.env` in the root first, if there is no `.env` in the root, it reads [env.development](../config/env.development).
+
+- Want to run back-end services locally, and to have front-end using the services.
+
+  This is the default setting, you **don't need to copy or modify** any env file. Just run `npm run services:start` and `npm run dev`. However, if it doesn't work and you have old `.env` that was copied from old `.env.example`, delete it.
+
+- Only want to work on front-end, and won't make any changes on the back-end.
+
+  ```
+  # 1. copy config/env.staging to .env.
+  # If you have old .env, it is good to delete it before copy to prevent overwrite error
+  cp config/env.staging .env
+  # 2. run front-end application
+  npm run dev
+  ```
+
+- Want to mix and match services between local and staging.
```

Comment:
Before or after this section we should mention our original API docs like `You can find the API docs at <link to API docs>`

Then go into the more specific use cases either here or in the doc:

```Our documentation regarding our new APIs are available at (API doc link here).

To run all services: npm run services: start

The option to run only selected services is also available by using the command: npm run services: start <service1> <service2>

Prior to running the commands. Please make sure the .env file is configured appropriately.

For example (use the auth example from line 19 and on here)